src/common/reactNativeProjectHelper.ts

```diff
@@ -17,11 +19,13 @@ export class ReactNativeProjectHelper {
     }
 
     /**
-     * Ensures that we are in a React Native project and then executes the operation
+     * Ensures that we are in a React Native project
      * Otherwise, displays an error message banner
-     * {operation} - a function that performs the expected operation
      */
     public isReactNativeProject(): Q.Promise<boolean> {
+        if (!fs.existsSync(path.join(this.workspaceRoot, "package.json"))) {
```

Comment:
Is this the right condition? I believe that running `react-native foo` requires you to be in the folder with the `node_modules/react-native` folder inside it, so if your folder structure looks something like

```
root
root/project/package.json
root/project/node_modules/react-native
root/site/<other content>
```

then `workspaceRoot` will point to `root`, but the react-native project is located at `root/project` and that's where the `package.json` must be, and where we should start `react-native` from.